### PR TITLE
Fix GH#20991: Chord symbols do not display correctly in parts when a custom score style is set

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: #20991

Fixes the scenario described at https://github.com/musescore/MuseScore/issues/20991#issuecomment-1890196996

### CODE REVIEW

**Review by cbjeukendrup — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
@oktophonie mind to review?

**Comment by Jojo-Schmitz** [comment]
@oktophonie mind to review?

**Comment by Jojo-Schmitz** [comment]
@oktophonie mind to review?

**Comment by Jojo-Schmitz** [comment]
Just came up once again in https://musescore.org/en/node/362569

**Comment by Jojo-Schmitz** [comment]
Let me know if and when a corresponging PR for 4.3.0 is wanted... that one-liner should be easy and harmless enough

File: src/engraving/dom/score.cpp [hunk 1]
```diff
@@ -178,6 +178,8 @@ Score::Score(MasterScore* parent, bool forcePartStyle /* = true */)
     m_masterScore = parent;
     if (DefaultStyle::defaultStyleForParts()) {
         m_style = *DefaultStyle::defaultStyleForParts();
+
+        checkChordList();
```
Comment (cbjeukendrup) [inline src/engraving/dom/score.cpp:182]: Actually, maybe it's preferable to move the `checkChordList()` calls out of this if/else statement, and put it just below `m_style.precomputeValues();`.

**Comment by cbjeukendrup** [comment]
And yes, a 4.3.0 version would seem good to me. Thanks!

**Comment by Jojo-Schmitz** [comment]
done ;-)

**Comment by Jojo-Schmitz** [comment]
Dang! The change doesn't seem to fix the issue :-(
Tested with this 
[ILikeTheFlowersLEAD.zip](https://github.com/musescore/MuseScore/files/14893824/ILikeTheFlowersLEAD.zip)

**Comment by Jojo-Schmitz** [comment]
Actually might be an entirely different issue: that score doesn#' have an issue with chord symbols in parts, but in the main score.

**Comment by cbjeukendrup** [comment]
@Jojo-Schmitz Could you summarise once more for Dmitry what is and isn't fixed by this PR?

**Comment by Jojo-Schmitz** [comment]
> And yes, a 4.3.0 version would seem good to me. Thanks!

See https://github.com/musescore/MuseScore/pull/22242#issuecomment-2060523246 :-(

**Comment by Jojo-Schmitz** [comment]
What is holding this from getting merged? At https://github.com/musescore/MuseScore/issues/20991#issuecomment-2145673229 there was just another confirmation that it solves the issue

**Comment by cbjeukendrup** [comment]
@DmitryArefiev @zacjansheski If one of you could give this your official approval, that would be great!

**Comment by zacjansheski** [comment]
Tested on MacOS 14, Windows 11, Ubuntu 22.04.3. Approved
https://github.com/musescore/MuseScore/issues/20991 FIXED

**Pull request merged**